### ISSUE TITLE
[Suggestion]Log Stick shows ProtectedAreas

### ISSUE THREAD

**Issue by WilliePlaysMC** [TotalFreedom/TotalFreedomMod#1641]
When I was online today, an op told me to unprotect their build. The only problem was when I asked him what was his build called on the protectarea list. He had no clue. Therefore, I am making the suggestion that if you use the logstick on a protectarea it shows you the name of the protectarea so you can remove it without clearing the whole list.

**Comment by DUFCLiam** [comment]
Agreed! This was @TomCreeperMC he asked me but an admin done it and I did not know what one it was.

**Comment by samennis1** [comment]
Great idea! I agree.
